setup.sh

```diff
@@ -20,6 +20,7 @@ echo ""
 ### Install .NET ###
 ####################
 echo "Installing .NET..."
+curl -L https://dot.net/v1/dotnet-install.sh | bash -e
```

Comment:
What is this for? Isn't this repeated in the line below?

Reply:
the line below install 5.0 sdk, this one makes sure that an LTS is also there. There are couple of other things before we can remove the LTS from the install script completely.